wandb/sdk/wandb_init.py

```diff
@@ -962,6 +962,7 @@ def init(
     monitor_gym: Optional[bool] = None,
     save_code: Optional[bool] = None,
     id: Optional[str] = None,
+    fork_from: Optional[Union[RunMoment, str]] = None,
```

Comment:
i suggest we don't expose the `RunMoment` for now

Reply:
done